src/main/java/seedu/address/model/profile/ProfileItem.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.model.profile;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Profile Item in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class ProfileItem {
+
+    // Identity fields
+    private final String title;
+    private final String type;
```

Comment:
type is category right? because it can only be either education, skill, or achievement should it be an enum instead?

Reply:
I was going for a primitive implementation for now, but since the types are fixed, I think enum is good solution too.